src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
-    <VersionPrefix>0.18.0-preview</VersionPrefix>
+    <VersionPrefix>1.1.0</VersionPrefix>
```

Comment:
The stable version should start from `1.0.0` here

Reply:
Sure